data/language/pl-PL.yml

```diff
@@ -2,7 +2,7 @@ header:
   locale: pl-PL
   english_name: Polish
   native_name: Polski
-  loco_original_id: 0
+  loco_original_id: 6
```

Comment:
Please revert this change. This refers to Locomotion's string tables, which has no entry for Polish. (Hence, it should fall back to English, 0.)